klarna-on-demand/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.klarna.ondemand">
 
+    <uses-permission android:name="android.permission.READ_PROFILE" />
```

Comment:
Don't forget we need to justify these in the docs, to assuage privacy weary users.